config/workflows/accessionWF.xml

```diff
@@ -31,25 +31,30 @@
     <label>Publish Metadata</label>
     <prereq>shelve</prereq>
   </process>
-  <process batch-limit="1" error-limit="5" name="provenance-metadata" sequence="9">
+  <process batch-limit="100" error-limit="10" name="update-doi" sequence="9">
```

Comment:
i have no idea what batch-limit does, but I expect that you have deliberately set it to 100 for this step and left it at 1 for the subsequent steps.

Reply:
I don't think it does anything, I just copied from a different step that had 100.